conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -99,10 +101,8 @@ def find_altered_packages(prefix: str | Path) -> dict[str, list[str]]:
     return altered_packages
 
 
-def display_health_checks(prefix: str, verbose: bool = False) -> None:
```

Comment:
Per CEP-8 we cannot delete functions outright, instead we need to deprecate them.